modules/auxiliary/scanner/portscan/ack.rb

```diff
@@ -27,6 +27,8 @@ def initialize
       OptString.new('PORTS', [true, "Ports to scan (e.g. 22-25,80,110-900)", "1-10000"]),
       OptInt.new('TIMEOUT', [true, "The reply read timeout in milliseconds", 500]),
       OptInt.new('BATCHSIZE', [true, "The number of hosts to scan per set", 256]),
+      OptInt.new('DELAY', [true, "The delay between connections, per thread, in milliseconds", 0]),
+      OptInt.new('JITTER', [true, "The delay jitter factor (maximum value by which to +/- DELAY) in milliseconds.", 0]),
```

Comment:
I admit it will be suboptimal to have _any_ option that is only honored sometimes, regardless of whether or not it is advanced.